ecommerce-demo/workspaces/demo/ldm.json

```diff
@@ -147,7 +147,7 @@
       },
       {
         "id": "monthlyinventory",
-        "title": "Monthlyinventory",
+        "title": "Monthly inventory",
         "description": "Monthlyinventory",
```

Comment:
Should we change `"Monthlyinventory"` → `"Monthly inventory"` as well?

Also, should we change `"Monthlyinventory"` → `"Monthly inventory"` at the following places as well, for the sake of consistency?

- https://github.com/gooddata/gooddata-public-demos/blob/5554bda5aecb596bb9685cb421b2a733358cf1ff/ecommerce-demo/workspaces/demo/ldm_v2.5.0.json#L143

- https://github.com/gooddata/gooddata-public-demos/blob/5554bda5aecb596bb9685cb421b2a733358cf1ff/ecommerce-demo/workspaces/demo/ldm_v2.5.0.json#L206